.changeset/dry-rules-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': patch
+---
+
+Fixes cacheOnDemandPages setting to only cache GET/HEAD requests by default
```

Comment:
Just a tiny suggestion, let me know what you guys think 😁
```suggestion
Enhanced caching behavior for the `cacheOnDemandPages` setting. Now, it explicitly caches only GET and HEAD requests by default.
```

Reply:
Thanks for the suggestion, I think it is good to add some more details, like you did. However we as maintainers unofficially agreed to use the present tense, so I suggest:

Enhances caching behavior for the `cacheOnDemandPages` setting. Now, it explicitly caches only GET and HEAD requests by default.